Dockerfile

```diff
@@ -20,16 +18,12 @@ RUN apk add --update \
         openssl-dev \
         unixodbc-dev \
         && \
-    pip install --upgrade \
-        pip \
-        setuptools \
-        virtualenv \
+    pip3 list --outdated --format=freeze | grep -v '^\-e' | cut -d = -f 1  | xargs -n1 pip3 install --no-cache-dir -U \
         && \
-    pip3 install --upgrade \
+    pip3 install --upgrade --no-cache-dir \
         pip \
```

Comment:
Does this need to be `pip3` now?

Reply:
I don't believe so, I think pip3 is just a command alias for the pip module. For example on my Mac:

`bblakely@bblakelyMAC ~ % pip3 list
Package    Version
---------- -------
pip        19.2.3
setuptools 41.2.0
six        1.15.0
wheel      0.33.1
WARNING: You are using pip version 19.2.3, however version 21.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
bblakely@bblakelyMAC ~ % python3 -m pip list
Package    Version
---------- -------
pip        19.2.3
setuptools 41.2.0
six        1.15.0
wheel      0.33.1
WARNING: You are using pip version 19.2.3, however version 21.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.`